src/main/java/ladder/controller/LadderController.java

```diff
@@ -55,4 +86,25 @@ private int readLadderHeight() {
         return inputView.requestLadderHeight();
     }
 
-}
+    private boolean response(Result result) {
+        String name = readName();
+
+        try {
+            return printGameResult(result, name);
+        } catch (IllegalArgumentException e) {
+            resultView.printErrorMessage(e.getMessage());
+            return response(result);
+        }
+    }
+
+    private boolean printGameResult(Result result, String name) {
+        if (name.equals(ALL)) {
```

Comment:
```suggestion
        if (ALL.equals(name)) {
```
이렇게하면 혹시나 모를 null pointer exception을 방지할 수 있습니다

Reply:
매개변수 name이 null일 가능성을 놓쳤습니다.
해당 사항 반영 하였습니다 !